### PR TITLE
Antisegment general

### PR DESCRIPTION
**Pull request by VargBurz**
some bugs with indents, but in result its ok

### CODE REVIEW

File: analytics/models/general_model.py [hunk 1]
```diff
@@ -23,6 +23,8 @@ def __init__(self):
             'convolve_max': 240,
             'convolve_min': 200,
             'WINDOW_SIZE': 240,
+	     'conv_del_min': 100,
```
Comment (rozetko) [inline analytics/models/general_model.py]: Let's fix indentation, please

File: analytics/models/general_model.py [hunk 2]
```diff
@@ -53,6 +55,20 @@ def do_fit(self, dataframe: pd.DataFrame, segments: list) -> None:
             convolve_data = scipy.signal.fftconvolve(labeled_data, self.model_gen)
             convolve_list.append(max(auto_convolve))
             convolve_list.append(max(convolve_data))
+				
+	del_conv_list = []
```
Comment (rozetko) [inline analytics/models/general_model.py]: And here

File: analytics/models/general_model.py [hunk 3]
```diff
@@ -63,6 +79,16 @@ def do_fit(self, dataframe: pd.DataFrame, segments: list) -> None:
             self.state['convolve_min'] = float(min(convolve_list))
         else:
             self.state['convolve_min'] = self.state['WINDOW_SIZE'] / 3
+		
+	if len(del_conv_list) > 0:
```
Comment (rozetko) [inline analytics/models/general_model.py]: And here :)

File: server/package.json [hunk 4]
```diff
@@ -38,7 +38,7 @@
     "es6-promise": "^4.2.4",
     "event-stream": "^3.3.4",
     "file-loader": "^1.1.11",
-    "grafana-datasource-kit": "^0.0.7",
+    "grafana-datasource-kit": ">=0.0.9",
```
Comment (jonyrock) [inline server/package.json]: vnezapno

**Pull request merged**